### PR TITLE
app-backup/mkstage4: version bump 0.2

### PR DESCRIPTION
**Pull request by TheChymera**
Package-Manager: Portage-2.3.83, Repoman-2.3.20
Signed-off-by: Horea Christian <chr@chymera.eu>

### CODE REVIEW

File: app-backup/mkstage4/mkstage4-0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (juippis) [inline app-backup/mkstage4/mkstage4-0.2.ebuild]: 7 :)

File: app-backup/mkstage4/mkstage4-0.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+DESCRIPTION="Bash Utility for Creating Stage 4 Tarballs"
+HOMEPAGE="https://github.com/TheChymera/mkstage4"
+SRC_URI="https://github.com/TheChymera/mkstage4/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND=""
```
Comment (juippis) [inline app-backup/mkstage4/mkstage4-0.2.ebuild]: These empty global variables can be removed.

File: app-backup/mkstage4/mkstage4-0.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+RDEPEND="app-shells/bash
+	app-arch/tar"
```
Comment (juippis) [inline app-backup/mkstage4/mkstage4-0.2.ebuild]: Please sort these, looks good otherwise!

**Comment by TheChymera** [comment]
@juippis the test failures seem to be unrelated to the package.

File: app-backup/mkstage4/mkstage4-0.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	app-arch/tar
+	app-shells/bash
+"
+
```
Comment (lucianposton) [inline app-backup/mkstage4/mkstage4-0.2.ebuild]: tar and bash are in default @system so you could just remove RDEPEND per https://devmanual.gentoo.org/general-concepts/dependencies/index.html#implicit-system-dependency

Comment (juippis) [reply app-backup/mkstage4/mkstage4-0.2.ebuild]: This is true, but some people want to add them to imply it's a simple bash script. As it's not a strict policy, I leave it up for the maintainer. I'd personally remove them.

**Pull request closed without merging**